### DISCUSSION TITLE
Texture different in AR and Modelviewer

### DISCUSSION THREAD

**Discussion by divclass1** [google/model-viewer#4777] [Q&A]
Hi,

When I upload a 3D model in the MV editor, my 3D model looks good. But when I use the Modelviewer script on my website, the textures changes. This also happens in AR. 

**2 questions**
1: What is the difference between the MV editor and the MV script on my website?
2: How can I solve this? 

Thank you!
![Modelviewer](https://github.com/google/model-viewer/assets/120512726/4fc59d04-2395-4eae-bf7d-85c07882f73b)
![Script on website and AR](https://github.com/google/model-viewer/assets/120512726/40392260-4ad1-4724-8240-947f19cd1782)

**Comment by danilokrulj** [comment]
Can you post the code you are using? Do you apply the textures in your model-viewer script or do you just import the model and it shows like this?

**Reply by divclass1** [reply]
I just use the modelviewer script on my site. I do use a 3D model with texture included. In the editor it looks good, but in on my website and AR it doesn't.

**Reply by elalish** [reply]
What version of MV are you using? Current is v3.5.0 - I'd guess you're on an old one with some bugs that have been fixed in the meanwhile.

**Reply by divclass1** [reply]
I import the script like this: https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js

**Reply by elalish** [reply]
That should work, but it's safer to use a more reliable CDN and pin your version, e.g. `https://ajax.googleapis.com/ajax/libs/model-viewer/3.5.0/model-viewer.min.js`
